src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.ColorSupplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private int getRandomProperties() {
+        int randomFigure = 20;

```

Comment:
lets make number 20 as constant field https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
what about this comment?